src/simd_128.h

```diff
@@ -56,32 +56,32 @@ typedef __m128i VecType;
 
 /* ============= Essential Operations for SSE w/ both u16 & u32 ============ */
 
-inline VecType LOAD(VecType* address)
+inline VecType LoadToReg(VecType* address)
```

Comment:
Why the `CamelCase`?

All the other functions are named in `snake_case`?

Also, we should not have a commit "rename things", you should do an interactive rebase and fix the naming directly in the commits that introduce these name (" SIMD 128: essential operations for SSE" and "SIMD 256: essential operations for AVX").